core-library/src/main/java/org/silverpeas/core/admin/persistence/UserRoleTable.java

```diff
@@ -288,167 +290,162 @@ public void removeUserRole(int id) throws AdminPersistenceException {
     updateRelation(DELETE_USERROLE, id);
   }
 
-  static final private String DELETE_USERROLE = "delete from ST_UserRole where id = ?";
+  private static final String DELETE_USERROLE = "delete from ST_UserRole where id = ?";
 
   /**
    * Tests if a user has a given role (not recursive).
    * @param userId
    * @param userRoleId
    * @return
-   * @throws AdminPersistenceException
+   * @throws SQLException
    */
-  public boolean isUserDirectlyInRole(int userId, int userRoleId) throws AdminPersistenceException {
+  public boolean isUserDirectlyInRole(int userId, int userRoleId) throws SQLException {
     int[] ids = new int[] { userId, userRoleId };
     Integer result = getInteger(SELECT_COUNT_USERROLE_USER_REL, ids);
     return result != null && result >= 1;
   }
 
-  static final private String SELECT_COUNT_USERROLE_USER_REL =
+  private static final String SELECT_COUNT_USERROLE_USER_REL =
       "select count(*) from ST_UserRole_User_Rel"
       + " where userId = ? and userRoleId = ?";
 
   /**
    * Add an user in a userRole.
    * @param userId
    * @param userRoleId
-   * @throws AdminPersistenceException
+   * @throws SQLException
+   * @throws AdminException
    */
-  public void addUserInUserRole(int userId, int userRoleId) throws AdminPersistenceException {
+  public void addUserInUserRole(int userId, int userRoleId) throws SQLException, AdminException {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Refactor this method to throw at most one checked exception instead of: java.sql.SQLException, org.silverpeas.core.admin.service.AdminException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1160)